### ISSUE TITLE
Feature request: Bluetooth BR/EDR support

### ISSUE THREAD

**Issue by jsmif** [whad-team/whad-client#47]
At defcon the "Exploiting Bluetooth From your car to the bank account" talk added new tooling for BR/EDR (https://github.com/sgxgsx/BlueToolkit). Under the premise of WHAD being _The One Tool To Rule Them All_, it'd be great if it added BR/EDR support, so that it could subsume their work, rather than continuing to need multiple tools in the BR/EDR space too.

I know it's probably not a high priority, but it's one of the most close protocols to the current feature set, so it seems like a natural target, so I figured it'd be worth making a placeholder ticket.

**Comment by RCayre** [comment]
Hi, thanks for the suggestion !

Bluetooth BR/EDR is definitely one of the protocol we plan to integrate in a next release, I mark the ticket as enhancement. It will be a lot of work and we will probably need time since the stack of complex and a lot of protocols need to be implemented and tested. 

If you are familiar with Bluetooth BR/EDR, what kind of hardware would you suggest to support? HCI and LMP (w Internablue enabled boards) seems to be a good start, but are you expecting specific tools?

**Comment by RCayre** [comment]
Yes, I agree. It could be interesting to check if it is possible to include it in node MCU firmware @virtualabs  ?

**Comment by virtualabs** [comment]
I've checked and Matheus used an ESP32-WROVER and not an ESP32-WROOM-32, that means the chips are different (ESP32-WROVER uses a D0WD-V3 while ESP32-WROOM-32 uses a D0WDQ6) and so the ROM could also be. Worst case scenario: we port Matheus' firmware to WHAD and it will only work with an ESP32-WROVER kit with Bluetooth classic only. Best case: we manage to do the same on an ESP32-WROOM-32 and it supports both Bluetooth Low Energy and Classic. 

In both cases we will need to update the protocol to add support for Bluetooth Classic with specific messages, but that's not really an issue.
